codechecker_lib/arg_handler.py

```diff
@@ -101,6 +103,40 @@ def handle_server(args):
 
     workspace = args.workspace
 
+    if (args.list or args.stop or args.stop_all) and \
+            not (args.list ^ args.stop ^ args.stop_all):
+        print("CodeChecker server: error: argument -l/--list and -s/--stop "
```

Comment:
I know this is not consistent at the moment, but we might want to avoid print and use LOG.error here.